### PR TITLE
OpenEBS: Updated with 2.11.0 latest chart

### PR DESCRIPTION
**Pull request by is-ashish**
This PR contains changes 

- URL changed in package.yaml for 2.11.0
- Annotations for partner, package-name in Chart.yaml
- Newly generated Chart.yaml.patch file
- Newly generate assets/, charts/ directories.

### CODE REVIEW

**Review by PennyScissors — CHANGES_REQUESTED** [review]
Hello @is-ashish, looks like something went wrong when you ran `make prepare` and it wasn't resolved. This PR is adding files that shouldn't be added (for example .orig and .rej files which are used for rebasing when a patch doesn't apply cleanly), and is removing files that shouldn't be removed app-readme.md, and questions.yaml. Also, all the required annotations and kubeConfig are missing from the Chart.yaml.

**Comment by is-ashish** [comment]
Hi @PennyScissors  getting following patching error. when i executed `make prepare` command.

```
INFO[0000] Pulling https://github.com/openebs/charts/releases/download/openebs-2.11.0/openebs-2.11.0.tgz from upstream into charts 
INFO[0003] Loading dependencies for chart               
INFO[0003] Found chart options for openebs-ndm in generated-changes/dependencies/openebs-ndm/dependency.yaml 
INFO[0003] Found chart options for localpv-provisioner in generated-changes/dependencies/localpv-provisioner/dependency.yaml 
INFO[0003] Found chart options for cstor in generated-changes/dependencies/cstor/dependency.yaml 
INFO[0003] Found chart options for jiva in generated-changes/dependencies/jiva/dependency.yaml 
INFO[0003] Found chart options for zfs-localpv in generated-changes/dependencies/zfs-localpv/dependency.yaml 
INFO[0003] Found chart options for lvm-localpv in generated-changes/dependencies/lvm-localpv/dependency.yaml 
INFO[0003] Pulling https://github.com/openebs/cstor-operators/releases/download/cstor-2.11.0/cstor-2.11.0.tgz from upstream into charts 
INFO[0004] Pulling https://github.com/openebs/jiva-operator/releases/download/jiva-2.11.0/jiva-2.11.0.tgz from upstream into charts 
INFO[0004] Pulling https://github.com/openebs/dynamic-localpv-provisioner/releases/download/localpv-provisioner-2.11.0/localpv-provisioner-2.11.0.tgz from upstream into charts 
INFO[0005] Pulling https://github.com/openebs/lvm-localpv/releases/download/lvm-localpv-0.7.0/lvm-localpv-0.7.0.tgz from upstream into charts 
INFO[0005] Pulling https://github.com/openebs/node-disk-manager/releases/download/openebs-ndm-1.6.0/openebs-ndm-1.6.0.tgz from upstream into charts 
INFO[0006] Pulling https://github.com/openebs/zfs-localpv/releases/download/zfs-localpv-1.9.0/zfs-localpv-1.9.0.tgz from upstream into charts 
INFO[0007] Updating chart metadata with dependencies    
WARN[0007] Detected 'apiVersion:v2' within Chart.yaml; these types of charts require additional testing 
INFO[0007] Applying changes from generated-changes      
INFO[0007] Applying: generated-changes/patch/Chart.yaml.patch 
ERRO[0007] 
patching file Chart.yaml
Hunk #1 FAILED at 17.
1 out of 1 hunk FAILED -- saving rejects to file Chart.yaml.rej 
FATA[0007] Encountered error while preparing main chart: Encountered error while trying to apply changes to charts: Unable to generate patch with error: exit status 1 
Makefile:7: recipe for target 'prepare' failed
make: *** [prepare] Error 1
```
Helm version: `Version:"v3.5.2"` and have created a new branch of `main-source` .

**Comment by PennyScissors** [comment]
Hello @is-ashish, when you run `make prepare PACKAGE={YOUR_PACKAGE_NAME}` and a patch for a file is present, we pull your chart into the `/packages/{YOUR_PACKAGE_DIR}/charts` directory and apply those patches. In some cases a patch may not apply cleanly and manual rebasing is necessary like in your case with. There seems to be a conflict in your `Chart.yaml` that is causing the patch to fail. You can use the generated `Chart.yaml.orig` and `Chart.yaml.rej` file to figure out what's the problem, and delete them afterwards.

**Comment by is-ashish** [comment]
Hi @PennyScissors can you please take a look PR ? have made changes as per suggested. Thanks

File: packages/openebs/generated-changes/patch/Chart.yaml.patch [hunk 1]
```diff
@@ -1,9 +1,12 @@
...
- sources:
- - https://github.com/openebs/openebs
- version: 1.12.3
+@@ -1,5 +1,9 @@
+ apiVersion: v2
+ appVersion: 2.11.0
++kubeVersion: '>=1.13.0-0'
 +annotations:
 +  catalog.cattle.io/certified: partner
 +  catalog.cattle.io/release-name: openebs
```
Comment (PennyScissors) [inline packages/openebs/generated-changes/patch/Chart.yaml.patch:9]: Please add the display name annotation for the name you want to be displayed in the UI. For example: `catalog.cattle.io/display-name: OpenEBS`. Do not modify this file directly, follow the steps to add it to the patch using the make commands.

**Review by PennyScissors — CHANGES_REQUESTED** [review]
Looks good overall, and it deploys fine within Rancher. Just need to address my other comment and add the app-readme.md, and questions.yaml files that were deleted. The questions.yaml file is optional, so it's up to you if you want to include that one or leave it out, but having it will result in a nicer UI to configure the chart rather than needing to modify the values yaml. More information on those files [here](https://github.com/rancher/partner-charts/tree/main-source#add-the-overlay-files-for-new-packages-only).

**Comment by is-ashish** [comment]
Hi @PennyScissors have added `app-readme.md`, and `questions.yaml` files. can you please take a look ? Thanks

**Review by PennyScissors — APPROVED** [review]

**Pull request merged**